flag/service/kubeconfig/resource/resource.go

```diff
@@ -0,0 +1,6 @@
+package resource
+
+// Resource is a data structure to hold Resource specific configuration flags.
+type Resource struct {
```

Comment:
Doesn't it make more sense to make it `KubeConfig.Secret.Namespace`? I wouldn't bake implementation details into flags.